src/tests/dbus-tests/test_90_lsm.py

```diff
@@ -0,0 +1,42 @@
+import os
+import dbus
+import storagedtestcase
+
+
+class StoragedLsmLocalTestCase(storagedtestcase.StoragedTestCase):
+    """
+    Test suit for org.freedesktop.UDisks2.Drive.LsmLocal interface.
+    """
+    _LED_CONTROL_METHOD_NAMES = ["TurnFaultLEDOn", "TurnFaultLEDOff",
+                                 "TurnIdentLEDOn", "TurnIdentLEDOff"]
+
+    def _get_dbus_drv_obj(self, dbus_blk_obj):
+        obj_path = self.get_property(dbus_blk_obj, '.Block', 'Drive')
+        return self.bus.get_object(self.iface_prefix, obj_path)
+
+    def test_led_ctrl(self):
+        """
+        Tests:
+         * Check the existence of LED control dbus methods.
+         * Expecting to get a "org.freedesktop.UDisks2.Error.NotSupported"
+           as we are running on LIO iscsi disks.
+        """
+        dbus_blk_obj = self.get_object('', '/block_devices/' +
```

Comment:
`StoragedTestCase.get_object` has changed in 5e074d1c2bc8a7dc06e5e5792291808e1ac45709 You need to remove the first parameter (the empty string).